lib/reporters.js

```diff
@@ -14,6 +15,17 @@ var defaultReporter = function (result) {
   });
   return out.join('\n');
 },
+// Default Partial Grunt reporter
+defaultPartialReporter = function (result) {
```

Comment:
Can this be avoided? It seems like the reporter shouldn't need to know about the wrapper. As long as its needed, won't it also break custom reporters?